Add members and description to gh frontpage (#2)
BiAPoL/.github: profile/README.md

# Bio-image analysis technology development group (BiAPoL)

> Welcome to the profile of the bio-image analysis technology development group (BiAPoL) at [Physics of Life](https://physics-of-life.tu-dresden.de/). We provide services and tools for the analysis of microscopy images in the life sciences. If you have any questions or need help with image analysis, feel free to contact us (see contact information below).

Reviewer: ```suggestion
Welcome to the profile of the bio-image analysis technology development group (BiAPoL) at [Physics of Life / TU Dresden](https://physics-of-life.tu-dresden.de/). We provide services and tools for the analysis of microscopy images in the life sciences. If you have any questions or need help with image analysis, feel free to contact us (see contact information below).
```

| <img src="./images/internship.png" alt="Consultation" style="width:50%"> | <p> <h2>**Internship/mini-sabbaticals Development** </h2> <br> **Stay with us to bolster your analysis and code proficiency** <ol> <li>2-4 weeks lab rotation for a focused bio-image analysis task</li> <li>Turn images into knowledge </li> </ol> <p>|
| <img src="./images/plugin.png" alt="Consultation" style="width:50%"> | <p> <h2>**Plugin/library development** </h2> <br> **Make your code sustainable and reusable** <ol> <li>[FAIRness](https://www.go-fair.org/fair-principles/)</li> <li>Napari plugins, Fiji plugins, Jupyter notebooks or Python libraries (i.e., PyPi)</li> </ol> <p>|
| <img src="./images/rent.png" alt="Consultation" style="width:50%"> | <p> <h2>**Rent-a-scientist** </h2> <br> **Pay us to support your work** <ol> <li>Organize workshops or other teaching events</li> <li>Get the help of a professional image data scientist for your project to make a leap forward.</li> </ol> <p>|

## Members

> We are currently three postodcs and one student working in the group:

Reviewer: ```suggestion
We are currently three postdocs and one student working in the group:
```

| | |
|---|---|
| ![image](./images/marcelo_cropped.png) | <p> <h2>**Marcelo Zoccoler** (Postdoc) </h2> <br> Interested in Image + Signal Processing, Fluorescence Lifetime-based Segmentation, Time-lapse Signal Classification, Machine Learning, Software and Plugin Development @zoccoler </p>|
| ![image](./images/johannes_cropped.png) | <p> <h2>**Johannes Soltwedel** (Postdoc) </h2> <br> Interested in segmentation, machine/deep learning, algorithms, teaching, surface meshes and geometry. </p> @jo-mueller|
| ![image](./images/stefan_cropped.png) | <p> <h2>**Stefan Hahmann** (Postdoc) </h2> <br> Interested in Research Software Engineering, Fiji Plugins, Feature Extraction, Deep Learning, Classification, Graphs, Open Source. </p> @stefanhahmann|
| ![image](./images/maleeha_cropped.png) | <p> <h2>**Maleeha Hassan** (Student) </h2> <br> Interested in Image Analysis, Machine Learning, meshes and cancer organoids. </p> @maleehahassan|

## Contact

> For more information on our current and previous members and our mission, visit our [group page](https://physics-of-life.tu-dresden.de/). For requests, please contact us via email (biapol at tu-dresden.de). For image analysis questions, please make sure to also check the image anylsis [forum](https://forum.image.sc/).

Reviewer: ```suggestion
For more information on our current and previous members and our mission, visit our [group page](https://physics-of-life.tu-dresden.de/research/core-groups/bio-image-analysis). For requests, please contact us via email (biapol (at) tu-dresden.de). For image analysis questions, please make sure to also check the image analysis [forum](https://forum.image.sc/).
```
